plugins/inputs/hddtemp/go-hddtemp/hddtemp.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
Because hddtemp is Linux only but reading the code a bit it actually runs on any platform.